sklearn/tests/test_pipeline.py

```diff
@@ -434,6 +435,7 @@ def test_score_samples_on_pipeline_without_score_samples():
     assert inner_msg in str(exec_info.value.__cause__)
 
 
+@pytest.mark.filterwarnings("ignore::FutureWarning")
```

Comment:
We will have to change this test in the future anyway: let's do this right away instead of hiding the warning.

```suggestion
```